src/legacy/core_plugins/kibana/public/discover/doc_table/components/_table_header.scss

```diff
@@ -1,7 +1,9 @@
+.kbnDocTableHeader button {
+  margin-left: 3px;
+}
```

Comment:
Margin is necessary, else there is non between the button, since in angular there was a whitespace between the buttons

Reply:
Can you just change the hard-coded 3px to `$euiSizeXS` which equals 4px?

Reply:
Of course!